src/VersionManager.cs

```diff
@@ -19,68 +20,162 @@ internal class VersionManager
                 {
-                    Console.Error.WriteLine(Output.Red($"Could not download DarkRift {version} - {tier} (.NET {platform}):\n\t{e.Message}"));
-                    return null;
+                    FileInfo fi = new FileInfo(paths[i]);
+                    versions[i] = new Version(fi.Name);
                 }
 
-                Console.WriteLine($"Extracting package...");
+                return versions;
+            }
+            catch
+            {
+                return new Version[0];
+            };
+        }
 
-                Directory.CreateDirectory(fullPath);
 
-                ZipFile.ExtractToDirectory(stagingPath, fullPath, true);
+        /// <summary>
+        /// Lists installed DarkRift versions on the console along with the documentation
+        /// </summary>
+        public static void ListInstalledVersions()
+        {
+            // Since the free version only supports .Net Framework I'm not adding support here
+            Version[] freeVersions = GetVersionDirectories(ServerTier.Free, ServerPlatform.Framework);
+
+            Version[] proFramework = GetVersionDirectories(ServerTier.Pro, ServerPlatform.Framework);
+            Version[] proCore = GetVersionDirectories(ServerTier.Pro, ServerPlatform.Core);
+
+            // Well, you gotta install it, you don't know what you are losing
+            if (freeVersions.Length == 0 && proFramework.Length == 0 && proCore.Length == 0)
+            {
+                Console.Error.WriteLine(Output.Red($"You don't have any version of DarkRift installed"));
+                return;
+            }
+
+            foreach (Version version in freeVersions)
+                PrintVersion(version, ServerTier.Free, ServerPlatform.Framework);
+            foreach (Version version in proFramework)
+                PrintVersion(version, ServerTier.Pro, ServerPlatform.Framework);
+            foreach (Version version in proCore)
+                PrintVersion(version, ServerTier.Pro, ServerPlatform.Core);
+        }
+
+        /// <summary>
+        /// Prints version information on the console
+        /// </summary>
+        /// <param name="version">The version number required.</param>
+        /// <param name="pro">Whether the pro version should be used.</param>
+        /// <param name="platform">Whether the .NET Standard build should be used.</param>
+        private static void PrintVersion(Version version, ServerTier tier, ServerPlatform platform)
+        {
+            string output = "";
 
-                Console.WriteLine(Output.Green($"Successfully downloaded package."));
+            // There's no free or pro in documentation
+            string documentation = string.Empty;
+            try
+            {
+                documentation = Directory.GetDirectories(Path.Combine(USER_DR_DIR, "documentation", version.ToString()))[0];
             }
+            catch { }
+
+            output += Output.Green($"{tier} {platform} version {version}");
```

Comment:
It would be good to follow the normal `DarkRift 2.5.0 - Free (.NET Framework)` format in this output. Also the output is informative rather than indicating a successful operation so shouldn't be coloured.